src/modules/Calendar/CalendarMonth.js

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+
+import '../../styles/CalendarStyles/CalendarMonth.css'
+
+import moment, {weekdaysShort} from "moment";
+import CalendarDay from "./CalendarDay";
+import {getSelectedDate, getSelectedMonth, getSelectedYear} from "./_selectors";
+import {selectDate} from "./_actionsDefinitions";
+import {connect} from "react-redux";
+
+
+class CalendarMonth extends React.Component {
+
+    generateWeeksArray(){
+        let firstDay = new Date(this.props.selectedYear, this.props.selectedMonth, 1);
+        let month = [];
+        let firstWeek = Array(7).fill('').map((v,i)=>{
+            return i < firstDay.getDay() ? 0 : Math.floor(i-firstDay.getDay() + 1).toString();
+        });
+        month.push(firstWeek);
+        let nextDay = 7 - firstDay.getDay() + 1;
+        let numOfWeeks = Math.ceil((moment(firstDay).daysInMonth() - nextDay) / 7);
+
+        for (let w=0; w < numOfWeeks; w++){
+            month.push(Array(7).fill().map((v, i) => {
+                let day = (w * 7) + nextDay + i;
+                return day < moment(firstDay).daysInMonth() ? day.toString() : "0";
+            }))
+        }
+        return month;
+    }
+
+    isDateToday(dateString) {
+        return new Date().getMonth() === this.props.selectedMonth
+            && new Date().getDate().toString() === dateString;
+    }
+
+    isDateSelected(dateString) {
+        return this.props.selectedDate && this.props.selectedDate.getDate().toString() === dateString;
+    }
+
+    renderWeek(arr, weekIndex) {
+        return (
+            <div key={arr[1]}>
```

Comment:
avoid unnecessary wrappers, I don't see the point of using one more `<div />` here